tests/package-title-dir-rtl-001/OPS/package.opf

```diff
@@ -1,18 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
-<metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
-  <dc:title dir="rtl">Package Title Direction RTL 001</dc:title>
-  <dc:date>2021-02-03</dc:date>
-  <dc:creator>Dave Cramer</dc:creator>
-  <dc:identifier id="pub-id">NOID</dc:identifier>
-  <meta property="dcterms:modified">2021-02-03T00:00:00Z</meta>
-  <dc:language>en</dc:language>
-</metadata>
-<manifest>
-  <item id="content_001" href="content_001.xhtml" media-type="application/xhtml+xml" />
-  <item id="nav" properties="nav" href="nav.xhtml" media-type="application/xhtml+xml" />
-</manifest>
-<spine>
-  <itemref idref="content_001" />
-</spine>
+    <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+        <dc:title dir="rtl" xml:lang="he">CSS: הרפתקה חדשה!</dc:title>
+        <dc:date>2021-06-24</dc:date>
+        <dc:creator>Ivan Herman</dc:creator>
+        <dc:identifier id="pub-id">package-title-dir-rtl-001</dc:identifier>
+        <meta property="dcterms:modified">2021-06-24T00:00:00Z</meta>
+        <dc:language>en</dc:language>
+        <dc:coverage>Internationalization Features</dc:coverage>
```

Comment:
As in #29, I recommend calling this just Internationalization.

Reply:
Moot by moving these into a separate PR (but will do there).